lib/hammer_cli_foreman_puppet/host.rb

```diff
@@ -1,47 +1,100 @@
-require 'hammer_cli_foreman_puppet/puppet_class'
-require 'hammer_cli_foreman/host'
+require 'hammer_cli_foreman_puppet/class'
+require 'hammer_cli_foreman_puppet/host'
+require 'hammer_cli_foreman_puppet/command_extensions/host.rb'
```

Comment:
is the file extension needed?
```suggestion
require 'hammer_cli_foreman_puppet/command_extensions/host'
```